.gitignore

```diff
@@ -98,6 +98,8 @@ publish/
 # NuGet Packages Directory
 ## TODO: If you have NuGet Package Restore enabled, uncomment the next line
 #packages/
+#/TabulateSmarterTestContentPackage/packages/
```

Comment:
Professional 2015 - The project was set up with the .sln file in the same directory as the project file, so I modified the .sln paths and moved the project file underneath the .sln so I could add the test project as a sibling.